src/model/image_info.js

```diff
@@ -360,6 +368,39 @@ export default class ImageInfo {
         });
     }
 
+    /**
+     * Checks that the imgData JSON response is valid, no exceptions etc.
+     *
+     * @private
+     * @param {Object} response the response object
+     * @memberof ImageInfo
+     * @return {Boolean} True if data is valid. Also shows dialogs with errors
+     */
+    validateImageInfo(response) {
+
+        if (response.ConcurrencyException) {
+            Ui.showModalMessage(`<p>Image is not currently viewable</p>
+                <pre>ConcurrencyException</pre>
+                <small>A pyramid of zoom levels is not available. Generation may be in progress if this is enabled on your server.</small>`, "OK");
```

Comment:
```suggestion
                <small>A pyramid of zoom levels is not available. Please contact your administrator, see https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#large-images.</small>`, "OK");
```

I realize this ^^^ is not ideal suggestion, but I am against making of false hopes - as we do not recommend pyramid generation to be enabled, we should not assume (as the present comment does) that "waiting it out" is a good/default option.

Reply:
Bit of a side track... but those docs don't describe how to enable/disable pyramid generation. Do we have install docs that recommend and describe how to disable pyramid generation? This is likely to have an effect on how likely it is for a particular server to have pyramids disabled, since the default is still to be enabled, right?
Also those docs should mention https://www.glencoesoftware.com/products/ngff-converter/ .

Unfortunately it's not possible for a client to know if pyramids are disabled, since this gives a SecurityViolation:
```
conn.getConfigService().getConfigValue("omero.server.nodedescriptor")
```

How about.
"A pyramid of zoom levels is not available. Please contact your administrator if this does not resolve itself. See <a href='https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#large-images'>Limitations: Large Images</a>"

Reply:
The property is actually `"omero.server.nodedescriptors"` and has been made user-visible since OMERO 5.6.6. Older versions of the server will definitely throw a SecurityViolation.

Note this is not necessarily the only way to disable the `PixelData` service e.g. you might have modified your templates manually but I think this is by far the option that is the most usable from an API perspective and the documentation should mostly likely document this workflow if any.

Reply:
@sbesson Thanks for catching my typo!
When connected to nightshade I get:
```
>>> conn.getConfigService().getConfigValue("omero.server.nodedescriptors")
''
```
Which would indicate that pyramid generation is enabled there?
For `outreach` server I get
```
'master:Blitz-0,Indexer-0,Processor-0,Storm,Tables-0'
```

Reply:
An empty value (default) means all services are enabled via configuration - see https://omero.readthedocs.io/en/stable/sysadmins/config.html#omero-server-nodedescriptors

Reply:
Great. Since clients can access this, I'll load it from iviewer so we can show a more useful error message for ConcurrencyException. It probably makes sense that we do something similar in webclient too, but most useful would be the Importer, since you need to know at import time if you're importing a Big Image without pyramids.